index.html

```diff
@@ -172,9 +172,15 @@ <h3>Create a github checklist</h3>
 <section id="I18N_Considerations">
 <h3>When and how to write an <em>Internationalization Considerations</em> section in your spec</h3>
 
-<p class="advisement">All additions of or changes to an <cite>Internationalization Considerations</cite> section MUST be reviewed by the Internationalization (i18n) WG.</p>
+<p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Ai18n_considerations_review" target="_blank">See related review comments.</a></p>
```

Comment:
The `t:...` label should use the id from specdev.

Since other section ids are lowercase, we should probably change the id of this section to something like `i18n_considerations` or `i18n_considerations_review`.

Reply:
Done

Reply:
Also fixed the label in the i18n-activity repo